docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,54 @@
+| `rbac.kargo.akuity.io/managed` | `ServiceAccount`, `Role`, `RoleBinding` | `"true"` | Indicates that the resource is programmatically managed by Kargo's API server. Resources with this annotation are created and maintained by Kargo and should not be modified manually. |
+| `rbac.kargo.akuity.io/claim.<name>` | `ServiceAccount` | Claim name as suffix | Maps an OIDC claim (e.g., `sub`, `email`, or `groups`) to a `ServiceAccount`, enabling user-to-ServiceAccount mappings. For more details, see the Access Control section here: [Operator Guide](../40-operator-guide/40-security/30-access-controls.md), [User Guide](../50-user-guide/50-security/20-access-controls/index.md). |
+
+## Labels
+
+| Label key | Target resource(s) | Possible values | Description |
+|-----------|--------------------|------------------|-------------|
+| `kargo.akuity.io/alias` | `Freight` | User-defined string | Friendly identifier for the `Freight` version. This label is automatically synced from the `alias` field of the `Freight` resource. Users should modify the `alias` field directly rather than editing this label, as the label exists primarily for convenience when using tools like `kubectl`. |
+| `kargo.akuity.io/project` | Project namespaces | `"true"` | Indicates that the namespace is eligible for adoption by a `Project` with the same name. This label is required for pre-created namespaces to be associated with a `Project`. For more details, see the [Working with Projects](./20-how-to-guides/20-working-with-projects.md#namespace-adoption) section. |
+| `kargo.akuity.io/freight-collection` | `Freight` | Collection name | System-assigned label that establishes a relationship between an `AnalysisRun` and a collection of `Freight`. |
+| `kargo.akuity.io/cred-type` | Credentials | `git`, `helm`, `image`, `generic` | Indicates the type of credentials stored in a `Secret` resource. This label is used to classify credentials for Git, Helm chart, container image repositories, or other generic purposes. For more details, see the [Managing Credentials](./50-security/30-managing-credentials.md#repository-credentials-as-secret-resources) section. |
+| `kargo.akuity.io/shard` | Internal resources | Shard ID | System-assigned label that indicates the logical cluster or controller shard ID. For `Warehouse` and `Stage` resources, users should modify the `spec.shard` field instead of this label. For `Promotions`, the system automatically copies the `Stage`'s `spec.shard` value to this label. |
+
+## Finalizers
+
+| Key | Target resource(s) | Possible values | Description |
+|-----|--------------------|------------------|-------------|
+| `kargo.akuity.io/finalizer` | All | N/A | Used by control plane to enforce proper cleanup. |
+
+
+:::info
+- If you're setting annotations via `kubectl`, use the `--overwrite` flag when updating existing annotations: 
+  ```
+  kubectl annotate stage my-stage kargo.akuity.io/refresh=$(date +%s) --overwrite
+  ```
+:::
```

Comment:
This is good advice, but feels like an awkward thing to tack on to the end of this page given that nowhere else on the page did we include any instructions for adding/modifying annotations via `kubectl`.

Let's cut this and we can always put it back in the future if we discover this is something we really _should_ have said after all.

```suggestion
```